test/unit/webdriver/app_test.py

```diff
@@ -23,84 +23,130 @@ class TestWebDriverApp(object):
     @httpretty.activate
     def test_install_app(self):
         driver = android_w3c_driver()
-        httpretty.register_uri(
-            httpretty.POST, appium_command('/session/1234567890/appium/device/install_app'), body='{"value": ""}'
-        )
         httpretty.register_uri(httpretty.POST, appium_command('/session/1234567890/execute/sync'), body='{"value": ""}')
         result = driver.install_app('path/to/app')
 
-        assert {'app': 'path/to/app'}, get_httpretty_request_body(httpretty.last_request())
```

Comment:
Like this test's assertion was wrong. I'm fixing this kind of error as well as part of https://github.com/appium/python-client/issues